packages/react/src/component-tests/IcSelect/IcSelect.cy.tsx

```diff
@@ -0,0 +1,2479 @@
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.checkShadowElVisible("ic-select", IC_MENU_LI);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_DOWN_ARROW);
+    cy.findShadowEl("ic-select", DATA_VALUE_ESPRESSO).should(
+      HAVE_ATTR,
+      ARIA_SELECTED
+    );
+    cy.findShadowEl("ic-select", "[data-value='espresso'] span").should(
+      HAVE_CLASS,
+      "check-icon"
+    );
+  });
+
+  it("should display a check next to the selected option and set aria-selected when a default value is applied", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+        value="espresso"
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.checkShadowElVisible("ic-select", IC_MENU_LI);
+    cy.clickOnShadowEl("ic-select", DATA_VALUE_ESPRESSO);
+    cy.findShadowEl("ic-select", DATA_VALUE_ESPRESSO);
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", DATA_VALUE_ESPRESSO).should(
+      HAVE_ATTR,
+      ARIA_SELECTED
+    );
+    // cy.findShadowEl("ic-select", DATA_VALUE_ESPRESSO).should(HAVE_CLASS, "check-icon");
+  });
+
+  it("should display clear button if the 'show-clear-button' prop is supplied and an option is selected", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+        showClearButton
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.checkShadowElVisible("ic-select", IC_MENU_LI);
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_DOWN_ARROW);
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_ENTER);
+    cy.findShadowEl("ic-select", ID_CLEAR_BUTTON).should(BE_VISIBLE);
+  });
+
+  it("should clear the input if the clear button is clicked", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+        showClearButton
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.checkShadowElVisible("ic-select", IC_MENU_LI);
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_DOWN_ARROW);
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_ENTER);
+    cy.clickOnShadowEl("ic-select", ID_CLEAR_BUTTON);
+    cy.findShadowEl("ic-select", IC_MENU_LI).should(NOT_BE_VISIBLE);
```

Comment:
```suggestion
    cy.get("ic-select").shadow().find(IC_MENU_LI).should(NOT_BE_VISIBLE);
```

changing to this fixed the failure for me

Reply:
This test doesn't currently fail for me in either browser or headless mode